### ISSUE TITLE
Release 0.0.5

### ISSUE THREAD

**Issue by Coffee2Bits** [jondavid-black/AaC#221]
This task is a placeholder/reminder to release a 0.0.5 version of AaC to pypi. This task should be one of the last performed for Sprint 2 to allow for the most-recent functionality to be pushed.

AC:
- [ ] Update AaC version to 0.0.5
- [ ] Run the deployment workflow

**Comment by cvchaparro** [comment]
Obviously I am aiming to finish the LSP stuff by the end of this sprint, but let's assume worst-case scenario for the moment - are we still planning to release in that case? I would think we'd want at least the minimum amount of LSP work that has been done to be in this release, no?

**Comment by Coffee2Bits** [comment]
> Obviously I am aiming to finish the LSP stuff by the end of this sprint, but let's assume worst-case scenario for the moment - are we still planning to release in that case? I would think we'd want at least the minimum amount of LSP work that has been done to be in this release, no?

Yep, we will release regardless; we have enough feature changes between 0.0.4's release and now that it's worth us sending out a new version. My goal is to get into a cadence where it doesn't matter much if we want X feature set in release Y because it'll have an opportunity within the next two weeks to get into another release.

**Comment by Coffee2Bits** [comment]
https://pypi.org/project/aac/0.0.5/
